wgsl/index.bs

```diff
@@ -4583,11 +4587,37 @@ See [[#sync-builtin-functions]].
   <tr algorithm="signed integer division">
     <td>|e1| : |T|<br>|e2| : |T|<br>|T| is [SIGNEDINTEGRAL]
     <td>|e1| `/` |e2| : |T|
-    <td>Signed integer division. [=Component-wise=] when |T| is a vector. (OpSDiv)
+    <td>Signed integer division. [=Component-wise=] when |T| is a vector.
+
+        In the scalar case, evaluates to:
+        * |e1|, when |e2| is zero.
+        * |e1|, when |e1| is the most negative value in |T|, and |e2| is -1.
+        * [=truncate=](|x|) otherwise, where |x| is the
+            real-valued quotient |e1|&nbsp;&div;&nbsp;|e2|.
+
+        Note:
+        The need to ensure truncation behaviour may require an implementation
+        to perform more operations than when computing an unsigned division.
+        Use unsigned division when both operands are known to have the same sign.
+
+        <!--
+               where Divisor = select(e2, 1, e2==0)
+               where MINIT = most negative value in |T|
+               select( truncate(e1/Divisor), MININT  , (MININT==e1) &&  (Divisor == -1))
```

Comment:
That's a bit unfortunate. It's a complex condition per element of division that's meant to handle a case that is extremely rare (unlike, say, division by zero).